### ISSUE TITLE
Surround sound

### ISSUE THREAD

**Issue by CoWayger** [benvanik/WebAL#1]
Why on the earth make OpenAL without surround sound? Why x,y,z coordinates when you end up with only stereo??

**Comment by Type1J** [comment]
I agree, but [binaural](https://en.wikipedia.org/wiki/Binaural_recording) output is probably the primary use.
